modules/network/private-dns-zones/srv/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Private DNS Zone SRV record",
+  "summary": "This module deploys a Private DNS Zone TXT record.",
```

Comment:
Should be "This module deploys a Private DNS Zone SRV record."

Reply:
Hey @lsnoddy, good catch. Fixed it.